assets/pages/digital-subscription-checkout/components/checkoutForm.jsx

```diff
@@ -105,8 +105,10 @@ function statesForCountry(country: Option<IsoCountry>): React$Node {
 
 function CheckoutForm(props: PropTypes) {
 
+  const errorHeading = props.submissionError === 'personal_details_incorrect' ? 'Failed to Create Subscription' :
```

Comment:
I'd try to move this to `errorReasons.js` if possible. Do we need a heading at all?

Reply:
Ooh, good point. We are currently using the `GeneralErrorMessage` component for a payment-related error so I suppose there are a few options
1) Use the `ErrorMessage` component for both. Then neither have a heading.
2) Put the logic to select an error heading in `generalErrorMessage.jsx` instead. (Is that any better?)
3) Use an `ErrorMessage` for form validation and a `GeneralErrorMessage` for payment errors
4) Rejig general error message to have the errorHeading be optional 
5) Something else?

Reply:
The way I'd go about this probably is to update the `appropriateErrorMessage()` function to return the body and a title maybe? So all the wording is in the same place

Reply:
I think the error reason `personal_details_incorrect` always implies an error message that is like `Please double check the name and contact details you provided and try again.` but not necessarily always the heading `Failed to Create Subscription`. If I can think of a heading that is more general but not so general as to be meaningless then yes, updating `appropriateErrorMessage()` seems like the best approach.